events/wc_events/wc_central_kalimdor_events.txt

```diff
@@ -4093,3 +4092,2089 @@ cen_kal.1053 = {
+	title = cen_kal.1069.name
+	desc = cen_kal.1069.desc
+	theme = hunt_activity
+	override_background = wc_background_tauren_tent_2
+	
+ 	left_portrait = {
+ 		character = root
+ 		animation = chaplain
+	}
+	
+	right_portrait = {
+ 		character = scope:shaman
+ 		animation = thinking
+	}
+	
+	immediate = { 
+		play_music_cue = "mx_cue_epic_sacral_moment" 
+		add_character_flag = has_totem
+		if = {
+			limit = { exists = cp:councillor_court_chaplain }
+			cp:councillor_court_chaplain = {
+				save_scope_as = shaman
+			}
+		}
+		if = {
+			limit = { NOT = { exists = scope:shaman } }
+			random_courtier = {
+				limit = { is_available_adult = yes }
+				save_scope_as = shaman
+			}
+		}
+	}
+	
+	option = {
+		name = cen_kal.1069.a
+		trigger_event = { id = cen_kal.1070 days = { 2 5 } }
+	}
+}
+cen_kal.1070 = {
+	type = character_event
+	title = cen_kal.1070.name
+	desc = cen_kal.1070.desc
+	theme = hunt_activity
+	override_background = wc_background_tauren_tent_2
+	
+ 	left_portrait = {
+ 		character = root
+ 		animation = personality_content
+	}
+	
+	right_portrait = {
+ 		character = scope:shaman
+ 		animation = toast
+	}
+	
+	option = {
+		name = cen_kal.1070.a
+		trigger_event = { id = cen_kal.1071 days = { 2 5 } }
+		stress_impact = {
+			trusting = minor_stress_impact_loss
+			paranoid = minor_stress_impact_gain
+		}
+		ai_chance = { base = 100 }
+	}
+	
+	option = {
+		name = cen_kal.1070.b
+		stress_impact = {
+			paranoid = minor_stress_impact_loss
+			trusting = minor_stress_impact_gain
+		}
+		remove_character_flag = has_totem
+		add_prestige = 350
+		custom_tooltip = ends_totem_questline_tt
+		ai_chance = { base = 0 }
+	}
+}
+cen_kal.1071 = {
+	type = character_event
+	title = cen_kal.1070.name
+	desc = cen_kal.1071.desc
+	theme = hunt_activity
+	override_background = wc_background_tauren_tent_2
+	
+ 	left_portrait = {
+ 		character = root
+ 		animation = personality_greedy
+	}
+	
+	option = {
+		name = cen_kal.1071.a
+		trigger_event = { id = cen_kal.1072 days = { 2 5 } }
+	}
+}
+cen_kal.1072 = {
+	type = character_event
+	title = cen_kal.1070.name
+	desc = cen_kal.1072.desc
+	theme = hunt_activity
+	override_background = wc_background_tauren_cave
+	
+ 	left_portrait = {
+ 		character = root
+	}
+	
+	right_portrait = {
+ 		character = scope:guide
+ 		animation = personality_content
+	}
+	
+	immediate = { 
+		create_character = {
+			location = root.location
+			culture=scope:shaman.culture
+			faith=scope:shaman.faith
+			dynasty=none
+			learning = { 14 22 }
+			age = { 35 45 }
+			gender_female_chance = 50
+			trait=diligent 
+			trait=calm 
+			trait=patient
+			after_creation = {
+				trigger_race_giving_no_gene_effect = yes
+				give_nickname = nick_brave
+			}
+			save_scope_as = guide
+		}
+	}
+	
+	option = {
+		name = cen_kal.1072.a
+		trigger_event = { id = cen_kal.1073 days = { 2 5 } }
+		if = { limit = { has_trait = impatient } add_stress = 15 }
+		if = { limit = { has_trait = patient } add_stress = -15 }
```

Comment:
```suggestion
		if = { limit = { has_trait = impatient } add_stress = 15 }
		else_if = { limit = { has_trait = patient } add_stress = -15 }
```

less checks since traits are opposite